### PR TITLE
docs(Storybook): more detailed doc for using our Storybook

### PR DESCRIPTION
**Pull request by asudoh**
#### Changelog

**New**

- Some explanation of how to use Storybook knobs/actions add-ons to see how to use our components.

### CODE REVIEW

File: docs/components.md [hunk 1]
```diff
@@ -23,11 +23,12 @@ We're actively looking to improve our Component documentation through generating
 add documentation around installing and using our components, in addition to
 covering common use-cases for each component with code examples.
 
-If you would like to help out, there are two components of contributing
+If you would like to help out, there are several steps of contributing
```
Comment (joshblack) [inline docs/components.md]: Maybe something like: 

```suggestion
If you would like to help out, there are several options to contribute
```

?

File: README.md [hunk 2]
```diff
@@ -10,7 +10,13 @@
 
 ### List of Available Components
 
-View available React Components [here](http://react.carbondesignsystem.com). Usage information is available when you click the blue **?** icon in the top right corner of the selected component.
+View available React Components [here](http://react.carbondesignsystem.com). You can see usage information in several ways:
+
+1. Clicking the blue **Show Info** icon in the top right corner of the selected component. You can see the list of available React props
+2. Clicking **STORY** tab at the bottom. You can see some example code there
```
Comment (joshblack) [inline README.md]:
```suggestion
2. Clicking the **STORY** tab at the bottom. This tab contains the code that shows how the component is being used
```

File: README.md [hunk 3]
```diff
@@ -10,7 +10,13 @@
...
+3. Clicking **KNOBS** tab at the bottom and changing values there. Most knobs are shown as something like `Button kind (kind)`, where `kind` is the name of React prop
```
Comment (joshblack) [inline README.md]: Maybe include `the` before the `**____** tab` lines just to help with how the sentence flows?

**Comment by asudoh** [comment]
Thank you for reviewing @joshblack - Made the updates you suggested.

**Pull request merged**